src/host/node_connections.h

```diff
@@ -305,11 +309,10 @@ namespace asynchost
       for (const auto node : local_queue)
       {
         LOG_DEBUG_FMT("reconnecting node {}", node);
-        auto s = find(node);
-
-        if (s)
+        auto s = outgoing.find(node);
```

Comment:
Only looking at the `outgoing` channels here because `find()` displays an undesirable log message if the channel is not found (and it probably won't be there as we are reconnecting).